elk/run.py

```diff
@@ -31,6 +34,20 @@
 )
 
 
+def fetch_git_hash() -> str:
+    try:
+        return (
+            subprocess.check_output(
+                ["git", "rev-parse", "HEAD"], cwd=Path(elk.__file__).parent.parent
+            )
+            .decode("ascii")
+            .strip()
+        )
+    except Exception as e:
+        print(f"Could not get git revision hash, stack trace: {e}")
+        return "ERROR_HASH_NOT_FOUND"
```

Comment:
Sorry to be so picky about this, but this really shouldn't be any kind of "error" because if the user installs from PyPI then it's expected that there won't be a git hash

Reply:
Oh ok. I had assumed that wouldn't be an issue as there aren't any users who are doing that now, or am I mistaken?

Reply:
Clément Dumas tried to do it. And I'd like to get more people using it soon